lib/terraforming/resource/ec2.rb

```diff
@@ -101,7 +101,7 @@ def monitoring_state(instance)
       end
 
       def instances
-        @client.describe_instances.reservations.map(&:instances).flatten.reject { |instance| instance.state.name == "terminated" }
+        @client.describe_instances.map(&:reservations).flatten.map(&:instances).flatten.reject { |instance| instance.state.name == "terminated" }
```

Comment:
__[rubocop]__
* Line is too long. [145/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)